fetcher/dockerhub.py

```diff
@@ -21,6 +22,12 @@ class dockerhubFetcher(tarFetcher):
     def __init__ (self, conffile, version, remote):
         super().__init__(conffile, version, remote)
         machine = runningDevice(conffile)
+
+        docker_version = "1.1.5"
+        if StrictVersion(getCurrentHostOSVersion(conffile)) < StrictVersion(docker_version):
```

Comment:
This looks like you're comparing a docker version with a host OS version which doesn't make any sense. I got it in the end but `docker_version` needs a better name. How about a `PRE_DOCKER_OS` constant?